deepchem/lightning/dc_lightning_module.py

```diff
@@ -0,0 +1,41 @@
+import pytorch_lightning as pl
```

Comment:
We shouldn't make a new top level folder for this feature. Instead, this should go in `deepchem/models/torch_models/lightning.py`

Reply:
I think we still need a dedicated folder for this, the new folder is at `deepchem/models/lightning`. Also I think it should not go under `torch_models` as it uses `torch_models` directly.

@rbharath what are your thoughts on this?

Reply:
I think `deepchem/models/lightning` is fine as a new folder